onnx/defs/nn/defs.cc

```diff
@@ -672,19 +680,31 @@ std::function<void(OpSchema&)> RoiPoolOpSchemaGenerator(const char* name) {
         "where N is the batch size, C is the number of "
         "channels, and H and W are the height and the "
         "width of the data.",
-        "T");
+        "T",
+        OpSchema::Single,
+        true,
+        1,
+        OpSchema::Differentiable);
     schema.Input(
         1,
         "rois",
         "RoIs (Regions of Interest) to pool over. Should "
         "be a 2-D tensor of shape (num_rois, 5) given as "
         "[[batch_id, x1, y1, x2, y2], ...].",
-        "T");
+        "T",
+        OpSchema::Single,
+        true,
+        1,
+        OpSchema::NonDifferentiable);
     schema.Output(
         0,
         "Y",
         "RoI pooled output 4-D tensor of shape (num_rois, channels, pooled_shape[0], pooled_shape[1]).",
-        "T");
+        "T",
+        OpSchema::Single,
+        true,
+        1,
+        OpSchema::NonDifferentiable);
```

Comment:
I guess it's differentiable? It's similar to one-to-one mapping.

Reply:
Yes, definitely. That was a mistake. Regarding the comment above (line 368): This is the indices output of MaxPool (not the actual pooling output). It's just the indices of the max value in the input window (an integer value if I understand correctly). Is it really differentiable?

Reply:
Yes, my understanding is that indexes are not differentiable but the output pooled from input is differentiable. :)